<launch>
  <node pkg="move_base" type="move_base" name="move_base" output="screen">
    <rosparam file="$(find final_odev)/config/costmap_common.yaml" command="load"/>
    <rosparam file="$(find final_odev)/config/global_costmap.yaml" command="load"/>
    <rosparam file="$(find final_odev)/config/local_costmap.yaml" command="load"/>
    <rosparam file="$(find final_odev)/config/dwa.yaml" command="load"/>
  </node>
</launch>

